off-chain/test/Faucet/Main.purs

```diff
@@ -0,0 +1,59 @@
+module Seath.Test.Faucet.Main (main) where
+
+import Contract.Prelude
+
+import Contract.Config
+  ( ContractParams
+  , defaultOgmiosWsConfig
+  , mkCtlBackendParams
+  , testnetConfig
+  )
+import Contract.Monad (launchAff_, runContractInEnv, withContractEnv)
+import Contract.Wallet (KeyWallet, withKeyWallet)
+import Data.UInt (fromInt) as UInt
+import Effect.Class.Console (log)
+import Node.FS.Aff (readdir)
+import Node.Path as Path
+import Seath.Test.Faucet.Contract as Faucet
+import Seath.Test.TestSetup (makeKeyWallet)
+
+main :: Effect Unit
+main = launchAff_ $ do
+  let
+    faucetPath = "./test/keys/faucet"
+    targetsPath = "./test/keys/seath_keys"
+    adaToPay = 500
+
+  (faucet /\ seathKeys) <- mkKeys faucetPath targetsPath
+
+  withContractEnv config $ \env -> do
+    for_ seathKeys $ \seathWallet ->
+      runContractInEnv env
+        (withKeyWallet faucet $ Faucet.payTo seathWallet adaToPay)
+  log "Funds seding end"
+
+mkKeys :: String -> String -> Aff (KeyWallet /\ Array KeyWallet)
+mkKeys faucetPath targetKeys = do
+  faucet <- makeFaucetWallet
+  seathKeys <- mekeSeathKeys
+  pure (faucet /\ seathKeys)
+  where
+
+  makeFaucetWallet = makeKeyWallet faucetPath
+
+  mekeSeathKeys = do
+    keyDirs <- readdir targetKeys
+    for keyDirs $ \keyDir -> makeKeyWallet $ Path.concat [ targetKeys, keyDir ]
+
+config :: ContractParams
+config = testnetConfig
```

Comment:
I see there are 3 different configurations sharing part of the parameters, should we just make our own config type and use it as a base for everyone?

Reply:
I would probably leave faucet's one as is, to be it close to "bare CTL", as it not really part of Seath, I'd say, but just a low-level tool to move funds between key wallets. I thought to use `cardano-cli` at first, but decided to go with CTL in the end.

Not sure even if it should be in `test`. Maybe make separate dir for it?

Reply:
Well from the library pov we needed it for the testnet only, so, leaving in tests is ok.

Reply:
And you are right, people can use the already implemented CTL mechanism for configuration.